survival/server/load_test_data.sh

```diff
@@ -0,0 +1,983 @@
+#!/bin/bash -e
```

Comment:
It probably makes more sense to have a smaller subset of this data loaded by default in the database through the docker entrypoint script.